invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/tombstone.html

```diff
@@ -8,17 +8,46 @@
 
 {%- set title = _("Tombstone") + " | " + config.THEME_SITENAME -%}
 
-{% block message %}
-  <h1><i class="fa fa-flash"></i> {{_('Gone')}}</h1>
+{%- block message %}
+  <h1><i class="fa fa-flash"></i> {{ _('Gone') }}</h1>
   <p>
-  {% trans sitename=config.THEME_SITENAME %}
-    The record  you are trying to access was removed from {{sitename}}. The
+  {%- trans sitename=config.THEME_SITENAME %}
+    The record  you are trying to access was removed from {{ sitename }}. The
     metadata of the record is kept for archival purposes.
-  {% endtrans %}
+  {%- endtrans %}
   </p>
-  {% if record and record.removal_reason %}
-  <p>
-    <strong>{{ _("Reason for removal:") }}</strong> {{record.removal_reason}}
-  </p>
-  {% endif %}
+  <div class="ui segment">
+  {%- if record and record.tombstone and record.tombstone.is_visible %}
+    {%- set removal_reason = record.tombstone.removal_reason.title.en or "None" %}
```

Comment:
```suggestion
    {%- set removal_reason = record.tombstone.removal_reason.title.en or "Reason unknown" %}
```
wouldn't it be better for user experience ?

Reply:
yes, that's definitely a clearer text :smile:

Reply:
i've wrapped the hard-coded string in a translation call, too.

however, i'm not sure how i would go about displaying the "correct" translation for the vocabulary title, if set?
right now the english value `record.tombstone.removal_reason.title.en` is hard-coded.

Reply:
found that this is usually done through the UI serializer; see the comment with new screenshots below.